src/WebJobs.Extensions.DurableTask/Options/TraceOptions.cs

```diff
@@ -37,6 +37,22 @@ public class TraceOptions
         /// </value>
         public bool TraceReplayEvents { get; set; }
 
+#if !FUNCTIONS_V1
+
+        /// <summary>
+        /// Gets or sets a flag indicating whether to enable distributed tracing.
+        /// The default value is true.
+        /// </summary>
+        public bool DistributedTracingEnabled { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a protocol for distributed Tracing.
+        /// Possible values are "HttpCorrelationProtocol" and "W3CTraceContext".
+        /// The default value is "HttpCorrelationProtocol".
+        /// </summary>
+        public string DistributedTracingProtocol { get; set; } = "HttpCorrelationProtocol";
+#endif
+
```

Comment:
nit: I think we want this whitespace before the #endif. 

I recommend building in both netstandard and net461 to see if you get stylecop errors in either. That should help diagnose whitespace issues in conditionally compiled segments.

Reply:
I've done the publishing. Both frameworks works.